framework/src/executioners/Transient.C

```diff
@@ -133,6 +133,10 @@ Transient::validParams()
   params.addParamNamesToGroup("start_time dtmin dtmax n_startup_steps trans_ss_check ss_check_tol "
                               "ss_tmin abort_on_solve_fail timestep_tolerance use_multiapp_dt",
                               "Advanced");
+  params.addParam<bool>("check_aux",
+                        false,
+                        "Whether to check the auxiliary system for convergence to steady-state. If "
+                        "false, then the nonlinear system is used.");
```

Comment:
need to add this parameter in the Advanced group, or maybe add a new group for steady-state detection and put this parameter along with few others in the group.

Reply:
I went ahead and added the parameter to the existing "Steady State Detection" group